extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -214,3 +215,48 @@ function get_colors( $name, $attrs, $property ) {
 
 	return $colors;
 }
+
+/**
+ * Render the given template in server-side.
+ *
+ * @param string $name  Template name, available in `./templates` folder.
+ * @param array  $data   Template data. Optional.
+ * @param bool   $print   Render template. True as default.
+ * @return false|string HTML markup or false.
+ */
+function render( $name, $data = array(), $print = true ) {
+	if ( ! strpos( $name, '.php' ) ) {
+		$name = $name . '.php';
+	}
+
+	$template_path = dirname( __FILE__ ) . '/templates/' . $name;
+
+	if ( ! file_exists( $template_path ) ) {
+		return '';
+	}
+
+	// Optionally provided an assoc array of data to pass to template
+	// and it will be extracted into variables.
+	if ( is_array( $data ) ) {
+		// phpcs:disable WordPress.PHP.DontExtract.extract_extract
+		extract( $data );
+		// phpcs:enable WordPress.PHP.DontExtract.extract_extract
```

Comment:
Instead of disable and enabling again, I think you could use `// phpcs:ignore` instead.

```suggestion
		extract( $data ); // phpcs:ignore WordPress.PHP.DontExtract.extract_extract
```

It may also be good to explain why you're disabling this rule here. You can add a comment at the end of the file like so:

```php
extract( $data ); // phpcs:ignore WordPress.PHP.DontExtract.extract_extract -- Disabled because reasons.
```

I think you can apply the same thing to other rules you've disabled below.

Reply:
> I think you can apply the same thing to other rules you've disabled below.

It gets really annoying for in the templates, @jeherve 

```php
'playerId' => $playerId, // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
'title'    => $title, // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
'link'     => $link, // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```
And we need to do this in all lines. I think applying it at the head and bottom of the file gets simpler.

Reply:
If you didn't use `extract`, would you run into those warnings here?